lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -1000,6 +845,104 @@ sub _resolve_workflow_for_build {
                 );
             }
         }
+        #Add gene category annotations to some output files from steps above. (e.g. determine which SNV affected genes are kinases, ion channels, etc.)
+        #AnnotateGenesByCategory - gene_category_<exome|wgs|wgs_exome>_<snv|indel>_result
+        my $annotate_genes_by_category_op = $self->annotate_genes_by_category_op($workflow);
+        $workflow->create_link(
+            source               => $converge_snv_indel_report_ops[-1],
```

Comment:
I think determining the best set of mapping and base qualities is a project in itself and I don't think we've done any formal evaluations in house based on simulated/validation data.
I think it's safe to pick highest cutoffs for base and mapping qualities and move ahead with those, with the current default processing profiles that would be a base-quality cutoff of 20 and mapping-quality cutoff of 30 which are not outrageous. If the processing profiles had incredibly high cutoffs the user would not see any variants downstream which is perhaps acceptable behavior. 
TL;DR: I like the idea of using the max, I want to make sure @malachig and @obigriffith are ok with this since this would affect some of the results we see.